Adds a collection to show ws usage with custom resource (#1)
PrestaShop/webservice-postman-examples: README.md

# Webservice Postman examples

> This repository is showing some Postman collections to illustrate how to use PrestaShop's webservices.

Reviewer: ```suggestion
This repository shows some Postman collections to illustrate how to use PrestaShop's webservices.
```

## What is Postman

Postman is a powerful collaboration platform that simplifies the process of developing, testing, documenting, and sharing APIs (Application Programming Interfaces) by providing a user-friendly interface and a comprehensive set of tools.

> As a developer, Postman can greatly assist you in various ways. It allows you to effortlessly send HTTP requests to APIs and receive responses, enabling you to test and debug your own APIs or interact with external APIs. Postman supports different request methods, request headers, query parameters, and request bodies, making it easy to explore API endpoints and their functionalities.

Reviewer: ```suggestion
As a developer, Postman can greatly assist you in various ways. It allows you to effortlessly send HTTP requests to APIs and receive responses, enabling you to test and debug your own APIs or interact with external APIs. Postman supports different request methods, request headers, query parameters, and request bodies, making it explore API endpoints and their functionalities.
```

More informations: [Official Postman website](https://www.postman.com/)

## Examples

### demo_custom_resource_collection.json

> This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).

Author: ```suggestion
This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here [extend webservice with the custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```

> This collection illustrates all the common CRUD actions on a custom resource (articles), from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here : [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).

Reviewer: ```suggestion
This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here  [extend webservice with the custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```

> This collection illustrates all the common CRUD actions on a custom resource (articles), from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here : [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).

Author: ```suggestion
This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```
